api-samples/downloads/downloads_overwrite/README.md

```diff
@@ -0,0 +1,13 @@
+# chrome.downloads
+
+A sample that demonstrates how to use the [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads/) API.
+
+## Overview
+
+In this sample, the [`chrome.downloads.onDeterminingFilename`](https://developer.chrome.com/docs/extensions/reference/downloads/#event-onDeterminingFilename) event is used to set conflict action to `overwrite` for all downloads.
```

Comment:
```suggestion
This sample uses the [`chrome.downloads.onDeterminingFilename`](https://developer.chrome.com/docs/extensions/reference/downloads/#event-onDeterminingFilename) handler to override target filenames for all downloads instead of adding ' (1)', ' (2)', etc..
```